crmprtd/process.py

```diff
@@ -59,16 +60,22 @@ def process(connection_string, sample_size, network, is_diagnostic=False):
     Session = sessionmaker(engine)
     sesh = Session()
 
-    observations = [
-        ob for ob in [align(sesh, row, is_diagnostic) for row in rows] if ob
-    ]
+    try:
```

Comment:
It's still a little unclear to me how to test this manually, so this is a catch all around both align() and insert() right now